### ISSUE TITLE
Feature Request: Add GeoMesa Support

### ISSUE THREAD

**Issue by harryprince** [harryprince/geospark#8]
According to @geoHeil information, it could be found at DataSystemsLab/GeoSpark#253 and any linked issues there. 
The first fundamental problem is the inconsistent representation of the geometry. i.e. both need to use org.locationtech.JTS some Geospark proprietary tools need to be merged to JTS first locationtech/jts#364 then, the namespace of geospark would need to switch from `vividsolutions` to `locationtech`

**Comment by kaushikCanada** [comment]
any progress on this?

**Comment by geoHeil** [comment]
Yes look at https://gitter.im/geospark-datasys/Lobby @jnh5y started to canalize efforts recently.
